libs/hdf-converters/src/converters-from-hdf/splunk/reverse-splunk-mapper.ts

```diff
@@ -231,9 +244,11 @@ export function createControlMapping(
       path: 'data.descriptions',
       transformer: (data: {label: string; data: string}[]) => {
         const descObjects: Record<string, string> = {};
-        data.forEach((item) => {
-          descObjects[item['label']] = item['data'];
-        });
+        if (Array.isArray(data)) {
```

Comment:
Will this still work for profiles that use tags for check, fix etc. They would just be the key value pair?

Reply:
Yes these are still preserved, you'll retain both tags and descriptions